ansible/roles/benchmark/tasks/run.yml

```diff
@@ -28,43 +28,50 @@
     block:
     - set_fact: users_per_sec_per_node="{{ (kcb_params | regex_search('--users-per-sec=(\S+)', '\\1') | join('') | int) / (node_count|int) }}"
     - debug: var=users_per_sec_per_node
+    - debug: msg="The users per second per node should not exceed 250, but is {{ users_per_sec_per_node }}. Add more nodes to create the load you asked for."
```

Comment:
should `- debug: msg="The users per second per node should not exceed 250, but is {{ users_per_sec_per_node }}. Add more nodes to create the load you asked for."` this be under the `failed_when` ? because right now it prints this when you run the playbook, which might be little confusing, even though this is a nitpick to some extent.

```TASK [benchmark : debug] ***************************************************************************************************************************
ok: [34.247.166.115] => {
    "users_per_sec_per_node": "100.0"
}

TASK [benchmark : debug] ***************************************************************************************************************************
ok: [34.247.166.115] => {
    "msg": "The users per second per node should not exceed 250, but is 100.0. Add more nodes to create the load you asked for."
}


```

Reply:
Thank you, I refined the message to be conditional depending on whether it exceeds the limit.